src/Parser/Druid/Restoration/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,166 @@
+            spell: SPELLS.ESSENCE_OF_GHANIR,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.TRANQUILITY_CAST,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.INNERVATE,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Manage your mana',
+      description: <Wrapper>Casting on targets who don't need healing or recklessly using inefficienct heals can result in running out of mana well before an encounter ends. Adapt your spell use to the situation, and as a rule of thumb try and keep your current mana percentage just above the bosses health percentage.</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: <Wrapper>Mana saved during <SpellLink id={SPELLS.INNERVATE.id} icon /></Wrapper>,
+            check: () => this.innervate.averageManaSavedSuggestionThresholds,
+            tooltip: `All spells cost no mana during Innervate, so take care to chain cast for its duration. Typically this means casting a Wild Growth, refreshing Efflorescence, and spamming Rejuvenation. It's also fine to Regrowth a target that is in immediate danger of dying.`,
+          }),
+          new Requirement({
+            name: <Wrapper>Use your <SpellLink id={SPELLS.CLEARCASTING_BUFF.id} icon /> procs</Wrapper>,
+            check: () => this.clearcasting.clearcastingUtilSuggestionThresholds,
+            tooltip: `This is the percentage of Clearcasting procs that you used. Regrowth is normally very expensive, but it's completely free with a Clearcasting proc. Use your procs in a timely fashion for a lot of free healing.`,
+          }),
+          new Requirement({
+            name: <Wrapper>Don't overuse <SpellLink id={SPELLS.REGROWTH.id} icon /></Wrapper>,
+            check: () => this.clearcasting.nonCCRegrowthsSuggestionThresholds,
+            tooltip: `This is the number of no-clearcasting Regrowths you cast per minute. Regrowth is very mana inefficient, and should only be used in emergency situations (and when you've already expended Swiftmend). Usually, you should rely on other healers in your raid to triage.`,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Use your defensive / emergency spells',
+      description: <Wrapper>Restoration Druids unfortunately do not have many tools to deal with burst damage, but you should take care to use the ones you have. The below percentages represent the percentage of time you kept each spell on cooldown.</Wrapper>,
+      requirements: () => {
+        return [
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.SWIFTMEND,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.IRONBARK,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.BARKSKIN,
+          }),
+        ];
+      },
+    }),
+    new PreparationRule(),
+  ];
+}
```

Comment:
I like the "Pick the right tools for the fight" which the holy paladin checklist has. We should probably include that aswell, for ToL/Cult (sotf LUL) and legendaries. Also give "error" if they are playing prosperity without t20.

Reply:
Good ideas, I wasn't sure what to do about the talent things. Probably group talent choice metrics together like that is good idea.